src/Common/src/System/Net/HttpKnownHeaderNames.TryGetHeaderName.cs

```diff
@@ -155,6 +155,7 @@ public static unsafe bool TryGetHeaderName(IntPtr buffer, int length, out string
                         case 'K': potentialHeader = KeepAlive; goto TryMatch;  // [K]eep-Alive
                         case 'S': potentialHeader = SetCookie; goto TryMatch;  // [S]et-Cookie
                         case 'U': potentialHeader = UserAgent; goto TryMatch;  // [U]ser-Agent
+                        case 'R': potentialHeader = RequestId; goto TryMatch;  // [R]equest-Id
```

Comment:
It doesn't hurt to have extra headers here, but I'm not familiar with this particular one.  Who sends this, and when?  Having this here is really only beneficial if it's coming down as part of a response.